android/app/build.gradle

```diff
@@ -155,6 +155,7 @@ android {
 }
 
 dependencies {
+    compile project(':react-native-code-push')
```

Comment:
In this way it will detect it's already "linked" even if `app` does not use it.

Reply:
shouldn't it be `implemenation` and not `compile`?

Reply:
Yeah but link puts `compile` still. I can change it manually however as this is the `app` it does not really matter.

Reply:
Ok, then I'll allow it 🙂